### DISCUSSION TITLE
how to custom flash.nvim jump labels?

### DISCUSSION THREAD

**Discussion by cStralpt** [LazyVim/LazyVim#1313] [Q&A] [answered]
i can't really see what letter that are highlighted by flash if i want to jump to certain location
![image](https://github.com/LazyVim/LazyVim/assets/95400822/164b5c51-05be-4a77-aac5-b66a6dbc1a06)

note: i use tokyonight theme

**Comment by dadav** [comment]
I think you can change the highlight groups via opts. This should be the relevant key: https://github.com/folke/flash.nvim/blob/6d76c5dee65181ab55cbdfb0760260e800d643f4/lua/flash/config.lua#L124

**Reply by cStralpt** [reply]
i wanted to customized that with my own colors, but for ttemporar solution i swittch my theme to "rose-pine".

**Comment by cStralpt** [comment, marked as answer]
i just replace all the default hi group of flash nvim by this config and it work perfectly:
```
local function setupCustomHighlightGroup()
	vim.api.nvim_command("hi clear FlashMatch")
	vim.api.nvim_command("hi clear FlashCurrent")
	vim.api.nvim_command("hi clear FlashLabel")

	vim.api.nvim_command("hi FlashMatch guibg=#4A47A3 guifg=#B8B5FF") -- Emerald background
	vim.api.nvim_command("hi FlashCurrent guibg=#456268 guifg=#D0E8F2")
	vim.api.nvim_command("hi FlashLabel guibg=#A25772 guifg=#EEF5FF")
end

return {
	"folke/flash.nvim",
	---@type Flash.Config
	opts = {
		rainbow = {
			enabled = true,
			shade = 5,
		},
		highlight = {
			backdrop = true,
			groups = {
				match = "FlashMatch",
				current = "FlashCurrent",
				backdrop = "FlashBackdrop",
				label = "FlashLabel",
			},
		},
	},
	config = function()
		setupCustomHighlightGroup()
	end,
}

```

**Reply by FloBurghardt** [reply]
thanks, i even like the colors you chose!
